examples/shell/shell_common/cmd_send.cpp

```diff
@@ -110,7 +110,6 @@ class MockAppDelegate : public Messaging::ExchangeDelegate
         streamer_printf(sout, "Response received: len=%u time=%.3fms\n", buffer->DataLength(),
                         static_cast<double>(transitTime) / 1000);
 
-        gExchangeCtx->Close();
```

Comment:
Same change needed in examples/shell/shell_common/cmd_ping.cpp

Reply:
Where exactly?  There is no `OnMessageReceived` implementation in `cmd_ping.cpp` anywhere that I can see.  `EchoClient::OnMessageReceived` is updated in this PR.  I feel like I'm missing something about this comment.

Reply:
Yes, I saw you made the change in EchoClient::OnMessageReceived.